Source/WTF/wtf/glib/GSpanExtras.h

```diff
@@ -102,6 +102,12 @@ inline std::span<const uint8_t> span(const GRefPtr<GVariant>& variant)
     return span(variant.get());
 }
 
+static inline std::span<char*> span(char **strv)
```

Comment:
Align the pointer asterisks to the left:

```suggestion
static inline std::span<char*> span(char** strv)
```

Reply:
Upsi-daisy!